products/cloudidentity/terraform.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Group: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_identity_groups_basic"
+        primary_resource_id: "cloud_identity_group_basic"
+        min_version: beta
+        vars:
+          id_group: "my-identity-group"
+        test_env_vars:
+          org_domain: :ORG_DOMAIN
+          cust_id: :CUST_ID
+      ### The full API doesn't seem to be implemented yet
+      # - !ruby/object:Provider::Terraform::Examples
+      #   name: "cloud_identity_groups_full"
+      #   primary_resource_id: "cloud_identity_group_full"
+      #   min_version: beta
+      #   vars:
+      #     id_group: "my-identity-group"
+      #   test_env_vars:
+      #     org_domain: :ORG_DOMAIN
+      #     cust_id: :CUST_ID
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      post_create: templates/terraform/post_create/set_computed_name.erb
+  GroupMembership: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_identity_group_membership"
+        primary_resource_id: "cloud_identity_group_membership_basic"
+        min_version: beta
+        vars:
+          id_group: "my-identity-group"
+        test_env_vars:
+          org_domain: :ORG_DOMAIN
+          cust_id: :CUST_ID
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_identity_group_membership_user"
+        primary_resource_id: "cloud_identity_group_membership_basic"
+        min_version: beta
+        vars:
+          id_group: "my-identity-group"
+        test_env_vars:
+          org_domain: :ORG_DOMAIN
+          cust_id: :CUST_ID
+          admin_user: :ADMIN_USER
```

Comment:
i don't believe so. i did this to separate that it has to be a user in admin console rather than a user in a project. maybe IDENTITY_USER might be more descriptive? i was trying to think on it last night, that's the best i could come up with.